composer.json

```diff
@@ -29,11 +29,10 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "php": ">=7",
+        "php": ">=7.4",
         "squizlabs/php_codesniffer": "^3.6.0",
-        "dealerdirect/phpcodesniffer-composer-installer": "~0.7.0",
-        "wp-coding-standards/wpcs": "^2.3",
-        "automattic/vipwpcs": "^2.2",
+        "dealerdirect/phpcodesniffer-composer-installer": "~1.0.0",
+        "automattic/vipwpcs": "dev-3.0/updates-for-wpcs-3.0",
```

Comment:
`3.0/updates-for-wpcs-3.0` has been merged 1 minute ago

https://github.com/Automattic/VIP-Coding-Standards/pull/779